config/typescript/tsconfig.base.json

```diff
@@ -26,6 +26,7 @@
       "esnext",
       "esnext.asynciterable"
     ],
-    "typeRoots": ["../../node_modules/@types", "./typings"]
+    "typeRoots": ["../../node_modules/@types", "./typings"],
+    "skipLibCheck": true
```

Comment:
This is necessary because sewing-kit will check the node_modules without it 

<img width="1487" alt="Screen Shot 2021-06-28 at 11 00 46 AM" src="https://user-images.githubusercontent.com/5274722/123662836-d382d380-d803-11eb-9c94-e14d18dbca4b.png">

Reply:
Every time this comes up we end up deciding that it's preferable to keep the lib check unless absolutely necessary. Why is this happening again and is there another way to fix the issue? Last time this happened it was due to pulling in multiple versions of React.

Reply:
There is an alternative to add `@types/react` to `resolutions` in `package.json` because `@types/react-reconciler` and `@types/react` are different version.

Reply:
Yes but removing this setting skips checking all node dependencies not just specify one's. Can we dedupe to use the same version of `@types/react`?

Reply:
We could deduplicate it. I tried with Olavo